travis-ymls/marathon-client.travis.yml

```diff
@@ -0,0 +1,24 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : marathon-client
+# Source Repo         : https://github.com/willroden/marathon-client
+# Travis Job Link     : https://travis-ci.com/github/santosh653/marathon-client
```

Comment:
Please update this link with latest build. @santosh653. Also squash the commits.